README.md

```diff
@@ -17,28 +10,32 @@ See features and benefits of [Algolia Search Extension for Magento](https://comm
 ![Magento 1.9.2](https://img.shields.io/badge/magento-1.9.2-blue.svg)
 ![Magento 1.9.3](https://img.shields.io/badge/magento-1.9.3-blue.svg)
 
-Documentation
---------------
+- **Auto-completion menu:** Offer End-Users immediate access to your whole catalog from the dropdown menu, whatever your number of categories or attributes.
```

Comment:
Sentence "... whatever your number of categories or attributes." misses the verb. I guess it should be "... whatever your number of categories or attributes is.", no?